pages/_app.js

```diff
@@ -1,8 +1,51 @@
+// Import external resources
+import { SWRConfig } from "swr";
+import { useImmer } from "use-immer";
+
+// Import internal resources
 import GlobalStyle from "@/styles";
 import Head from "next/head";
-import { SWRConfig } from "swr";
+import Layout from "@/components/Layout";
 
 export default function App({ Component, pageProps }) {
+  const [cartItems, updateCartItems] = useImmer([]);
+  const [cartTotal, setCartTotal] = useImmer(0);
```

Comment:
isn't the cartTotal a derived state from your cartItems? You could either use the length of the array if its supposed to display the number of items, or you can use the reduce function that you wrote to calculate the total price.

Reply:
- [x] Fixed was the leftover of last user story 👍